Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -454,6 +454,22 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.8,
 )
 
+class UpgradeWorkflow_JMENano(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Nano' in step:
+            stepDict[stepName][k] = merge([{'--customise_commands': '"from PhysicsTools.NanoAOD.custom_jme_cff import PrepJMECustomNanoAOD_MC; PrepJMECustomNanoAOD_MC(process)"'}, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        return '2017' in key or '2018' in key
```

Comment:
Right now this adds a `_JMENano` workflow for every single fragment. I would highly recommend requiring `fragment=="TTbar_13"` in the condition, unless all of those workflows are really needed.

Reply:
Done, thanks.